gui/src/test/java/org/verapdf/cli/commands/VeraCliArgParserTest.java

```diff
@@ -263,74 +263,6 @@ public final void testLogPassedSuccessOption() {
                 .logPassed());
     }
 
-    /**
-     * Test method for
-     * {@link org.verapdf.cli.commands.VeraCliArgParser#extractFeatures()}.
-     */
-    @Test
```

Comment:
for all tests in this file: you should not remove tests. Tests can be removed only if some logic has been removed